src/sequelize/sequelize/sequelize-service.ts

```diff
@@ -37,9 +38,9 @@ function getValidationOnlyOptions(options: SequelizeOptions): SequelizeOptions {
  * Determines models from value
  */
 export function getModels(
-  arg: Array<ModelCtor | string>,
+  arg: Array<ModelCtor<Model> | string>,
```

Comment:
Can you explain why you've add Model explicitly here? I'm asking since it is already set as default in ModelCtor definition

Reply:
Good catch: it and its siblings are legacy from another failed attempt that I missed in the cleanup phase.